src/routes/(app)/form/[id]/+page.server.ts

```
@@ -47,14 +48,30 @@ export const actions = {
 		const htmlContent = `Message envoyé par formulaire sur le site ${ORIGIN}<br> Adresse de contact : ${
 			parsed.from ? parsed.from : 'non-renseigné'
 		}  <br> Message envoyé :<br> ${parsedMessageHtml}`;
-		await collections.emailNotifications.insertOne({
-			_id: new ObjectId(),
-			createdAt: new Date(),
-			updatedAt: new Date(),
-			subject: parsed.subject,
-			htmlContent: htmlContent,
-			dest: parsed.target || SMTP_USER
-		});
+		const content = `Message envoyé par formulaire sur le site ${ORIGIN} Adresse de contact : ${
+			parsed.from ? parsed.from : 'non-renseigné'
+		}   Message envoyé : ${parsed.content}`;
+		if (contactForm) {
```

you should throw an error early (404) if `contactForm` doesn't exist